DataFormats/PatCandidates/interface/Jet.h

```diff
@@ -173,6 +173,9 @@ namespace pat {
                                     const unsigned int& set = 0) const {
       return correctedJet(level, flavor, set).p4();
     };
+    /// Scale energy and correspondingly adjust raw jec factors
+    void scaleEnergy(double fScale) override { scaleEnergy(fScale, "scaled"); };
+    void scaleEnergy(double fScale, std::string level);
```

Comment:
```static const& std::string```

Reply:
const std::string& level -> yes
why static? This method modifies the object.

Reply:
sorry, no ```static``` necessary